main.cpp

```diff
@@ -67,29 +67,38 @@ namespace Example
 #include <string>
 
 
-#define PASSENGER_COEFF 0.0000005
-#define CAR_ON_MOTORWAY_COEFF 0.0000002
-#define MOTORBIKE_ON_MOTORWAY_COEFF 0.0000006
-#define LORRY_ON_MOTORWAY_COEFF 0.0000006
+#define PASSENGER_COEFF 0.5
+#define CAR_ON_MOTORWAY_COEFF 0.2
+#define MOTORBIKE_ON_MOTORWAY_COEFF 0.6
+#define LORRY_ON_MOTORWAY_COEFF 0.1
 
 struct Car
 {
-    Car(std::string name_, double speed_, int passengers_) : name(name_), speed(speed_), passengers(passengers_) 
+    Car(std::string name_, double speed_, int passengers_) : name(name_), speed(speed_), passengers(passengers_)
     {
         for(size_t i = 0; i < passengers; ++i)
```

Comment:
`passengers` triggers a warning because of its type.   
- [ ] correct the type to clear the warning.